### DISCUSSION TITLE
Export option for PublishPress settings

### DISCUSSION THREAD

**Discussion by stevejburge** [publishpress/PublishPress-Planner#1268] [Feature Requests]
Good idea:

> When you setup a dev/local or test environnement, it would be super fun to be able to export all the hard work to stage/production :)
> What do you think ? Is an "export settings" (notifications workflow, status, base settings, ...) planned ?

**Comment by tazeverywhere** [comment]
👍

**Comment by CalebWoodbridge** [comment]
This would be very useful! I've just installed PublishPress on a dev environment to look at getting it set up for an existing site, and this would make life a lot easier.

**Comment by stevejburge** [comment]
Thanks @CalebWoodbridge. We'll take a look at getting this done

**Comment by samjco** [comment]
Any update on the export option?

**Comment by stevejburge** [comment]
@samjco Sorry, not yet
